### ISSUE TITLE
Nova 4 Compatability

### ISSUE THREAD

**Issue by sn-seanh** [CaddyDz/NovaPreviewResource#6]
Hi Guys,

Is this package compatible with Nova 4?

Thanks,

**Comment by CaddyDz** [comment]
Yes, since [this commit](https://github.com/CaddyDz/NovaPreviewResource/commit/909fd3f77fd7a17f52ecaa7505ae416cd1caffcc) 

Are you having any problems with Nova 4?
